src/d3a/d3a_core/global_objects.py

```diff
@@ -16,18 +16,35 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
 from d3a.models.strategy.external_strategies import ExternalMixin
-from d3a.d3a_core.util import get_market_maker_rate_from_config
+from d3a.d3a_core.util import get_market_maker_rate_from_config, \
+    find_object_of_same_weekday_and_time
+from d3a.models.strategy.infinite_bus import InfiniteBusStrategy
 
 
 class GlobalStatistics:
 
     def __init__(self, root_area):
         self.area_stats_tree_dict = {}
         self.root_area = root_area
+        self.feed_in_tariff = None
 
-    def update(self):
+    def update(self, current_market_slot=None):
         if self.root_area.current_market:
             self._create_grid_tree_dict(self.root_area, self.area_stats_tree_dict)
+        if current_market_slot is not None:
+            self._buffer_feed_in_tariff(self.root_area, current_market_slot)
+
+    def _buffer_feed_in_tariff(self, area, current_market_slot):
+        """
+        This simplified recursion is sufficient as the infinite bus is expected to be in the
+        uppermost level of the tree
+        """
+        for child in area.children:
+            if isinstance(child.strategy, InfiniteBusStrategy):
+                self.feed_in_tariff = \
```

Comment:
I would rename this to `_current_feed_in_tariff` and expose a `feed_in_tariff_rate` property in order to have a clean API in this class, because I predict that it will grow quite soon.